src/Microsoft.ML.Transforms/RffTransform.cs

```diff
@@ -1,51 +1,47 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
>Licensed [](start = 3, length = 8)

Why are you doing this Ivan?

Reply:
What can I say? I promise to be better, and not to do this again.

---
In reply to: [222843945](https://github.com/dotnet/machinelearning/pull/1122#discussion_r222843945) [](ancestors = 222843945)

Reply:
Ok, Fine, I forgive you this time, but better watch yourself!

---
In reply to: [222846814](https://github.com/dotnet/machinelearning/pull/1122#discussion_r222846814) [](ancestors = 222846814,222843945)

Reply:
:joy: 

---
In reply to: [222846868](https://github.com/dotnet/machinelearning/pull/1122#discussion_r222846868) [](ancestors = 222846868,222846814,222843945)